### PR TITLE
61 single product page

### PR DESCRIPTION
**Pull request by liliumorion**

### CODE REVIEW

File: src/components/ProductCard/index.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ function ProductCard({ title, location, listingType, imageUrl, category }) {
           </span>
         </div>
       </div>
-    </div>
+    </Link>
```
Comment (farouk26) [inline src/components/ProductCard/index.jsx:33]: Replace the `</Link>` with `</div>`

**Review by farouk26 — CHANGES_REQUESTED** [review]
Greetings @liliumorion 

After testing the branch from my side , there are a few changes you need to make 

in ProductCard component which is at `src/components/ProductCard/index.jsx ` , replace the `</Link>` with `</div>` tag , you'll find it in `Line 33`  
remove the unnecessary unused imports when finished 
Additionally , run this command `npm test -- -u` to update your test files , after doing that run `git add .` and then commit your changes 

Finally , fix your prettier code's format in all products page and check if your component has any missing format ... you can do that by holding `Ctrl + Shift + P` , and then type `Format Document` in the search and press `Enter` , this will fix your code format according to Prettier's rules , once you do all that don't forget to git add everything and then commit your changes , lets hope this fixes your failed checks in this PR

**Review by farouk26 — APPROVED** [review]
Everything seems to be fixed now , thank you for all the hard work ! , Approving changes

**Review by riadhmouamnia — APPROVED** [review]
## Approval review for Single Product Page ✅:

**Reviewer:** @riadhmouamnia

The "Single Product Page" looks fantastic! Your effort and dedication to this task are evident in the results. I appreciate your commitment to our project.

**Positive Aspects:**

1. **Product Details Card**: The ProductDetailsCard Component you created is not only well-designed but also responsive. It's crucial that users can access and view product details on any device, and you've ensured a seamless experience.

2. **Related Products Slider**: The addition of a slider for related products is an excellent touch. This feature greatly enhances the user experience by allowing easy navigation through related items.

3. **Data Fetching**: You've successfully fetched the appropriate data and displayed it on the page. This is a vital aspect of this task, and you've executed it perfectly.

**Feedback:**

I couldn't find any issues or concerns with your work. You've covered all aspects of the task, and the results are impressive. Keep up the excellent work, and thank you for your dedication to the project.

Once you've addressed any potential feedback from others, this is certainly ready for merging. Great job!

**Pull request merged**